qchem/tests/test_particle_number.py

```diff
@@ -55,7 +55,7 @@ def test_particle_number(orbitals, coeffs_ref, ops_ref):
     r"""Tests the correctness of the :math:`\hat{S}_z` observable built by the
     function `'spin_z'`.
     """
-    n = qchem.hf.particle_number(orbitals)
+    n = qchem.particle_number.particle_number(orbitals)
```

Comment:
This import format is temporary, in the future all functions will be available with `qml.qchem.name`.